addons/binding/org.openhab.binding.globalcache/src/main/java/org/openhab/binding/globalcache/GlobalCacheBindingConstants.java

```diff
@@ -37,11 +37,12 @@
     public final static ThingTypeUID THING_TYPE_ITACH_FLEX = new ThingTypeUID(BINDING_ID, "itachFlex");
     public final static ThingTypeUID THING_TYPE_GC_100_06 = new ThingTypeUID(BINDING_ID, "gc100_06");
     public final static ThingTypeUID THING_TYPE_GC_100_12 = new ThingTypeUID(BINDING_ID, "gc100_12");
+    public final static ThingTypeUID THING_TYPE_ZMOTE = new ThingTypeUID(BINDING_ID, "zmote");
```

Comment:
Sure.  I'm a bit of a git newb, so I may have a question or two along the way.  :-)